vignettes/lab_grading.Rmd

```diff
@@ -0,0 +1,395 @@
+
+From these SOC values the following terms criteria is implemented in {admiral} 
+
+From SOC = “Investigations" there are 21 CTCAE v4.0 Terms:
+
+  + Activated partial thromboplastin time prolonged
+  + Alanine aminotransferase increased
+  + Alkaline phosphatase increased
+  + Aspartate aminotransferase increased
+  + Blood bilirubin increased
+  + CD4 lymphocytes decreased
+  + Cholesterol high
+  + CPK increased
+  + Creatinine increased
+  + Fibrinogen decreased
+  + GGT increased
+  + Haptoglobin decreased
+  + Hemoglobin increased
+  + INR increased
+  + Lipase increased
+  + Lymphocyte count decreased
+  + Lymphocyte count increased
+  + Neutrophil count decreased
+  + Platelet count decreased
+  + Serum amylase increased
+  + White blood cell decreased
+
+From the SOC =  “Metabolism and nutrition disorders” there are 14 CTCAE v4.0 Terms:
+
+  + Hypercalcemia
+  + Hyperglycemia
+  + Hyperkalemia
+  + Hypermagnesemia
+  + Hypernatremia
+  + Hypertriglyceridemia
+  + Hyperuricemia
+  + Hypoalbuminemia
+  + Hypocalcemia
+  + Hypoglycemia
+  + Hypokalemia
+  + Hypomagnesemia
+  + Hyponatremia
+  + Hypophosphatemia
+
+From the SOC =  “Blood and lymphatic system disorders” there are 2 CTCAE v4.0 Terms:
+
+  + Anemia
+  + Leukocytosis
+
+## Updates made to TERM
+
+For terms "Hypocalcemia" and "Hypercalcemia" the criteria is provided for Calcium and Ionized Calcium,
+therefore {admiral} created a row for each in the metadata, this is noted in the COMMENT variable of
+the metadata:
+
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(
+  atoxgr_criteria_ctcv4,
+  display_vars = vars(TERM, COMMENT),
+  filter = str_detect(TERM, 'calcemia')
+)
+```
+
+Similarly, there is criteria applicable to Fasting Glucose as well as non-Fasting Glucose for "Hyperglycemia"
+so again this was split into 2 rows, and noted in the COMMENT variable. Note "Hypoglycemia" does not require to
+be split into 2 rows:
+
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(
+  atoxgr_criteria_ctcv4,
+  display_vars = vars(TERM, COMMENT),
+  filter = str_detect(TERM, 'glycemia')
+)
+```
+
+## Assumptions made when grading
+
+For term "INR Increased" there is the following criteria:
+
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(
+  atoxgr_criteria_ctcv4,
+  display_vars = vars(TERM, Grade_1),
+  filter = str_detect(TERM, 'INR')
+)
+```
+
+{admiral} assumed worst case and used both parts of the criteria for grading, so comparing
+lab value against ULN and also BASE. The decision made was put in the `COMMENT` field.
+
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(
+  atoxgr_criteria_ctcv4,
+  display_vars = vars(TERM, COMMENT),
+  filter = str_detect(TERM, 'INR')
+)
+```
+
+
+For TERM "Hyperuricemia", the criteria for Grade 1 and Grade 3 is the same with respect
+to the lab value, so worse case is assumed as grade 3. The decision made was 
+put in the `COMMENT` field.
+
+
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(
```

Comment:
Do you mind putting in `<br>` to help with readability?  

![image](https://user-images.githubusercontent.com/10111024/187248963-526ea8f1-43c8-4598-9708-5acb998d8a91.png)

Reply:
done, I added a break before and after each call to the dataset_vignette